misc.php

```diff
@@ -79,3 +79,19 @@ function action_wpcom_vip_verify_string() {
 		}
 	}
 }, 10, 1 );
+
+/**
+ * Hooks pre_ping to stop any pinging from happening,
+ * unless `WPCOM_VIP_DO_PINGS` is set to `true`.
+ *
+ * @param array $post_links The URLs to be pinged
+ */
+function wpcom_vip_pre_ping( $post_links ) {
+	if ( ! defined( 'WPCOM_VIP_DO_PINGS' ) || true !== WPCOM_VIP_DO_PINGS ) {
+		return;
```

Comment:
We want to empty out the array here, when `WPCOM_VIP_DO_PINGS` is _not_ `true`. Then below, return `$post_links`. 

Currently, this will not work since we're returning void here, and then clearing the array and returning nothing down below, so it effectively disables pinging even when the constant is set to `true`.